modules/programs/rofi.nix

```diff
@@ -280,7 +280,7 @@ in
     };
 
     configPath = mkOption {
-      default = ".config/rofi/config";
+      default = "${config.xdg.configHome}/rofi/config";
```

Comment:
I think you'll want to add something like
```nix
defaultText = "$XDG_CONFIG_HOME/rofi/config";
```
to avoid having to rebuild the documentation whenever `config.xdg.configHome` changes.

Reply:
Good point! I committed suggested change.